packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.Props.ts

```diff
@@ -75,4 +75,12 @@ export interface ITooltipHostProps extends React.HTMLAttributes<HTMLDivElement |
    * Optional class name to apply to tooltip host.
    */
   hostClassName?: string;
+
+  /**
+   * Manual way to display tooltip when the element is not directly focusable but an ancestor is,
+   * so it can pass down that state via this property.
+   * It honors overflow rules and keyboard and mouse interaction, so this just adds another interaction way
+   * delegated on another element.
+   */
+  isFocused?: boolean;
```

Comment:
I'm using `TooltipHost` because I want to use the overflow mode that displays the tooltip only when content overflows (you can see it in my screenshots, tooltip won't happen for other items like spec.md).
This is why the prop is named `isFocused`, because I don't want to force visibility, just want to force calculation of whether there's overflow to actually display the tooltip. I'm open to better name suggestions, but not sold to any of yours.